sentry-rails/lib/sentry/rails/tracing/active_support_subscriber.rb

```diff
@@ -0,0 +1,63 @@
+require "sentry/rails/tracing/abstract_subscriber"
+
+module Sentry
+  module Rails
+    module Tracing
+      class ActiveSupportSubscriber < AbstractSubscriber
+        READ_EVENT_NAMES = %w[
+          cache_read.active_support
+        ].freeze
+
+        WRITE_EVENT_NAMES = %w[
+          cache_write.active_support
+          cache_increment.active_support
+          cache_decrement.active_support
+        ].freeze
+
+        REMOVE_EVENT_NAMES = %w[
+          cache_delete.active_support
+        ].freeze
+
+        FLUSH_EVENT_NAMES = %w[
+          cache_prune.active_support
+        ].freeze
+
+        EVENT_NAMES = READ_EVENT_NAMES + WRITE_EVENT_NAMES + REMOVE_EVENT_NAMES + FLUSH_EVENT_NAMES
+
+        SPAN_ORIGIN = "auto.cache.rails".freeze
+
+        def self.subscribe!
+          subscribe_to_event(EVENT_NAMES) do |event_name, duration, payload|
+            record_on_current_span(
+              op: operation_name(event_name),
+              origin: SPAN_ORIGIN,
+              start_timestamp: payload[START_TIMESTAMP_NAME],
+              description: payload[:service],
```

Comment:
You're absolutely right. That's a bad leftover from activestorage instrumentation! I'll have a look at this one tomorrow.

Reply:
since CI is also failing in many places, I can take this over if you wish but you will need to give me maintainer push access
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests

Reply:
Be my guest! I'm happy to resolve the errors in here and in #2383 (Also from me/us :) ) 

I've pushed initial changes, but I am not running a full test suite - I'll get that up and running locally as well.

Any way to accept CI runs from this PR automatically? Instead of manual approval each time I do changes :)

Reply:
One thing - Before I go around pure anarchy in terms of `make_basic_app`. With the current set up I am unable to change the Rails.cache store to a memory store, because the block is yielded from an initializer - After initializing the rails app and cache store (Filestore by default).

I can either change the store for all `make_basic_app` setup (Which shouldn't affect much), or we can introduce a new parameter to `make_basic_app(cache_store:)`, but that seems like a bad direction to go.

The issue in specs is, that the Prune method, only works for memorystore.

When that is settled, I'll write specs for all of the other Rails.cache.* methods

Reply:
> Any way to accept CI runs from this PR automatically? Instead of manual approval each time I do changes :)

sadly not, corporate compliance :)

> I can either change the store for all make_basic_app setup (Which shouldn't affect much)

I _think_ nothing else is using/assuming filestore so you can go ahead with changing the default. In case something breaks, we can generalize `make_basic_app.`

Reply:
It's been merged, but for Rails main branch. How do we proceed here, as the "improved" solution is not merged back to "old" Rails versions?

We could just subscribe for the events per definition, and just support the ones, that the users Rails version have?

That wouldn't fix coverage percentage though (Until 8.0 releases, and coverage is run against Rails 8)

Reply:
@sl0thentr0py I have granted you write access for the fork (By transfering the forked repo to my own GitHub instead of work). That should do it, if you had any ideas here!

Reply:
> We could just subscribe for the events per definition, and just support the ones, that the users Rails version have?

Yes, just support what's possible, else just no-op/silently fail.

> That wouldn't fix coverage percentage though (Until 8.0 releases, and coverage is run against Rails 8)

I'll take care of the annoying stuff like coverage, don't worry about it too much. A couple of lines here and there that only run for some versions are ok to be left uncovered since we support such large version matrices and our codecov setup is not taking into account everything.

Reply:
Unless we want to introduce Redis or Memcached in the testing, which is a separate container for the GitHub Actions workflow.

Reply:
@sl0thentr0py Do we need anything else here except internal QA/processing from you guys?
We're looking forward to having this merged soon!

Reply:
No worries!

Reply:
Any update @sl0thentr0py? If you need anything from me, I'm happy to implement to save you guys time.

Reply:
@frederikspang hey hey, sorry for keeping you waiting, I'm gonna help you wrap this up as Neel is still too busy with other tasks :) I'll test it out locally and do a code review, and in the meantime could you resolve conflicts with the CHANGELOG file? 🙏🏻

Reply:
@solnic Done. Do you want me to squash commits or will you "Squash & Merge" from GitHub?
There may be some broken specs as well - If you allow them to run, I have a chance to take a look at them!

Reply:
> @solnic Done. Do you want me to squash commits or will you "Squash & Merge" from GitHub? There may be some broken specs as well - If you allow them to run, I have a chance to take a look at them!

I'll squash/merge, thanks.